grails-app/controllers/mama/ng/scheduler/MessageController.groovy

```diff
@@ -0,0 +1,110 @@
+package mama.ng.scheduler
+
+import grails.converters.JSON
+
+class MessageController {
+
+    static allowedMethods = [save: "POST", update: "PUT", delete: "DELETE"]
+
+    def cronParserService
```

Comment:
Oh, right, that creates an instance of the service. No need to do CronParserService cronParserService